gulpfile.js

```diff
@@ -150,37 +147,34 @@ gulp.task('lint-scss', () => {
  *
  */
 gulp.task('import-changelog-file', () => {
-  let url = "https://api.github.com/repos/youseedk/dna/releases";
-  if(githubClientId && githubClientSecret) {
+  let url = 'https://api.github.com/repos/youseedk/dna/releases';
+  if (githubClientId && githubClientSecret) {
     url = `${url}?client_id=${githubClientId}&client_secret=${githubClientSecret}`;
   }
-  return request(
-    {
-      url: url,
-      method: "get",
-      headers: {
-        "user-agent": "User-Agent"
-      }
-    },
-    function(error, response, body) {
-      if(error) {
-        console.error(error);
-      }
-    }
-  )
-  .pipe(source('github-releases.json'))
-  .pipe(gulp.dest(`${paths.tokensSource}/generated`));
-})
-
+  return request({
+        url: url,
+        method: 'get',
+        headers: {
+          'user-agent': 'User-Agent'
+        }
+      },
+      function (error) {
+        if (error) {
+          console.error(error);
+        }
+      })
+    .pipe(source('github-releases.json'))
+    .pipe(gulp.dest(`${paths.tokensSource}/generated`));
+});
 
 /**
-
+ *
  *	Convert colors.json file to .scss-file
  *
  */
-gulp.task('jsonToScss', () => {
+gulp.task('json-to-scss', () => {
   return gulp.

```

Comment:
the reasons why this sections fucks op with indentation is because of the dot. The dot should be moved to next line so that line becomes `.src` then it makes sense to indent it with 2 spaces.